CentaurusInfra/global-resource-scheduler
# Implemented site resource deduction after scheduling

**What type of PR is this?**
> /kind feature

**What this PR does / why we need it**: 
This PR proposed the deduction of resource from scheduler cache after scheduling a pod

**Which issue(s) this PR fixes**: 
Fixes #252 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
NONE

**How to test?**: 
(1) Run the following script 
~/go/src/k8s.io/arktos$ ./hack/globalscheduler/globalscheduler-up.sh
~/go/src/k8s.io/arktos$ cd ~/go/src/k8s.io/arktos/globalscheduler/test/yaml
$ kubectl apply -f globalscheduler/test/yaml/sample_6_clusters.yaml
$ kubectl apply -f sample_6_clusters.yaml
$kubectl apply -f sample_2_schedulers.yaml  
$kubectl apply -f sample_2_distributors.yaml   
$kubectl apply -f sample_2_dispatchers.yaml
$kubectl apply -f sample_6_pods.yaml
$kubectl get pods
(2) check openstack and log files at /tmp
